src/routes/projects/[projectId]/cases/[testCaseId]/+page.svelte

```
@@ -917,3 +950,55 @@ ${testCase.expectedResult || 'See test case for details'}`;
 		prefillDescription={jiraModalDescription}
 	/>
 {/if}
+
+{#if showDeleteConfirm}
+	<!-- svelte-ignore a11y_no_noninteractive_element_interactions -->
```

ESLint

[error] svelte-ignore comment is used, but not warned

(svelte/no-unused-svelte-ignore)